tsconfig.json

```diff
@@ -1,16 +1,13 @@
 {
+  "extends": "@tsconfig/recommended/tsconfig.json",
```

Comment:
Bringing in this base recommended config.  It includes several of the options removed below.